src/main/java/com/gamebros/purepazaak/controller/DeckController.java

```diff
@@ -0,0 +1,76 @@
+package com.gamebros.purepazaak.controller;
+
+import java.awt.Container;
+import java.util.ArrayList;
+
+import org.newdawn.slick.GameContainer;
+import org.newdawn.slick.Graphics;
+
+import com.gamebros.purepazaak.Match;
+import com.gamebros.purepazaak.entity.Card;
+import com.gamebros.purepazaak.enumerable.PlayerEnum;
+import com.gamebros.purepazaak.event.CardClickedEvent;
+import com.gamebros.purepazaak.event.ContextualEventInterface;
+import com.gamebros.purepazaak.listener.ActionListenerInterface;
+import com.gamebros.purepazaak.view.DeckView;
+
+public class DeckController extends AbstractController implements ActionListenerInterface {
+  protected GameContainer container;
+
+  protected ArrayList<Card> deck;
+
+  protected DeckView deckView;
+
+  protected Match match;
+
+  public DeckController(
+      GameContainer container,
+      ArrayList<Card> deck,
+      Match match
+  ) {
+    this.container = container;
+    this.deck = deck;
+    this.match = match;
+    this.init();
+  }
+
+  public void render(Graphics graphics) {
+    try {
+      this.deckView.render(graphics);
+    } catch (Exception e) {
+      return;
+    }
+  }
+
+  protected void init() {
+    this.deckView = new DeckView(
+        this.deck,
+        20,
+        480,
+        380,
+        100
```

Comment:
Encapsulate width, height, and position with a Value Object and set it as a constant on this class.

`this.deckView = new DeckView(this.deck, VIEW_COORDINATES);`

Reply:
Actually, what about using more concrete classes for encapsulating the size and coordinates for each view?

```
public class PlayerOneDeckView extends DeckView {
  public PlayerOneDeckView(this.deck) {
    // Use constants for the size and coordinates of the view.
  }
}
```